src/frontend/src/lib/components/ui/Tag.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+	export let ariaHidden = false;
+</script>
+
+<span
+	aria-hidden={ariaHidden ? 'true' : 'false'}
```

Comment:
Isn't Svelte mapping boolean to string?

`aria-hidden={true}` isn't parsed as `aria-hidden="true"` in the DOM?

Reply:
right! changed